src/components/workflow/workflow-utils/format.js

```diff
@@ -1,3 +1,8 @@
+// Helper to format a number, removing unnecessary trailing zeros (e.g., 1.50 -> 1.5, 2.00 -> 2)
+export function formatNumber(num) {
```

Comment:
I think we should not do this because 2B for the user was confusing and one thought it was a postcode i think we should keep 2 point decimal.